themes/CodeMeta-Pyramids/layouts/partials/footer.html

```diff
@@ -28,10 +29,15 @@
         </li>
         {{ end }}
       {{ end }}
+      {{ with .OutputFormats.Get "RSS" }}
+      <li>
+          <a href="{{ .RelPermalink }}" aria-label="RSS feed for {{ page.Title }}"><i class="bi bi-rss"></i> RSS</a>
+      </li>
+      {{ end }}
       </ul>
     </nav>
 	  <div class="copyright pull-right">
-	    <img src="{{"img/cc-by.svg" | relURL }}"/>
+	    <img aria-label="CC-By license badge" src="{{"img/cc-by.svg" | relURL }}" alt="Licensed CC-BY"/>
```

Comment:
```suggestion
	    <img src="{{"img/cc-by.svg" | relURL }}" alt="Licensed CC-BY"/>
```

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label says:

> If you [...] define an alt attribute for an [<img>](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/img), [...] aria-label is not necessary. However, if an aria-label attribute is present, it will take precedence over [...] the image's alt